beacon_node/beacon_chain/src/test_utils.rs

```diff
@@ -3245,96 +3250,49 @@ pub enum NumBlobs {
     None,
 }
 
+macro_rules! add_blob_transactions {
+    ($message:expr, $payload_type:ty, $num_blobs:expr, $rng:expr, $fork_name:expr) => {{
+        let num_blobs = match $num_blobs {
+            NumBlobs::Random => $rng.random_range(1..=2),
```

Comment:
Might be nice to pull these into `const`s?

Reply:
Yep, updated.

For some reasons I thought I did 0..1, maybe i did change it to 1..=2 😅